### PR TITLE
Add python container with seaborn

### PR DESCRIPTION
**Pull request by FerriolCalvet**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [X] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [X] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [X] LABEL base_image  
 - [X] LABEL version
 - [X] LABEL software.version  
 - [X] LABEL about.summary  
 - [X] LABEL about.home  
 - [X] LABEL about.license   
 - [X] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [X] LABEL about.documentation   
  - [X] LABEL about.license_file
  - [X] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: seaborn/0.12.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+################## BASE IMAGE ######################
+
+FROM python:3.10.8
+
+################## METADATA ######################
+
+LABEL base_image="python:3.10.8"
+LABEL version="2"
```
Comment (osallou) [inline seaborn/0.12.2/Dockerfile]: this is first release, should be version 1

File: seaborn/0.12.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+LABEL about.summary="Python library for plotting."
+LABEL software="seaborn"
+LABEL software.version="0.12.2"
+LABEL about.tags="implemented-in::python"
+LABEL about.home="https://seaborn.pydata.org"
+LABEL about.software="https://seaborn.pydata.org"
+LABEL about.documentation="https://seaborn.pydata.org"
+LABEL about.license="BSD-3-Clause"
+LABEL about.license_file="https://github.com/mwaskom/seaborn/LICENSE.md"
+LABEL extra.identifiers.biotools="seaborn, pandas, numpy, matplotlib"
```
Comment (osallou) [inline seaborn/0.12.2/Dockerfile]: should be a single tool link to a biotools identifier. In this case I do not find any biotools entry for seaborn, so should be left empty

**Review by osallou — CHANGES_REQUESTED** [review]

File: seaborn/0.12.2/Dockerfile (hunk 2 above)
Comment (FerriolCalvet) [reply seaborn/0.12.2/Dockerfile:18]: I am leaving pandas and python here, is it fine?

Comment (FerriolCalvet) [reply seaborn/0.12.2/Dockerfile:18]: or there should not be anything?

Comment (FerriolCalvet) [reply seaborn/0.12.2/Dockerfile]: Sorry I had misunderstood what should be here.
Both comments should be fixed now.
Thank you for the super fast review!

**Review by osallou — APPROVED** [review]

**Comment by osallou** [comment]
@FerriolCalvet merged, thanks

container will soon be available

**Comment by FerriolCalvet** [comment]
Hi @osallou do you know when will this container be available?
In fact there is already a seaborn 0.7.1 container available in biocontainers, but it was not present here in this repo.
[https://biocontainers.pro/tools/seaborn](https://biocontainers.pro/tools/seaborn)
I do not know if this might be an issue for automatically triggering the generation of this new container.
Thank you!

**Comment by osallou** [comment]
it is available:
https://hub.docker.com/r/biocontainers/seaborn/tags

**Comment by osallou** [comment]
but registry not up-to-date

**Comment by FerriolCalvet** [comment]
ah okay great, thank you!

**Comment by osallou** [comment]
this is a registry issue with dockerhub, seems we hit some rate limits with our api requests.... we need to patch code to handle this

**Pull request merged**